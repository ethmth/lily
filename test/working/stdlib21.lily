

let a:: int = [74,48,20,2,10,84,5,9,11,24,91,2,71,64,63,80,28,30,58,11,44,87,22,54,74,10,55,28,46,29,10,5072,34,26,25,8,51,13,30,35,8,50,65,6,16,2,2178,35,13,14,23,3,26,90,86,25,56,91,13,92,25,37,570,69,98,95,45,47,29,86,28,73,44,46,65,84,96,24,12,72,68,93,57,92,52,5,2,85,63,56,55,12,85,77,39]

print(a)

a.mergesort()

print(a)

print(min(a))
print(max(a))