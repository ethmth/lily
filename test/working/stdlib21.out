[74, 48, 20, 2, 10, 84, 5, 9, 11, 24, 91, 2, 71, 64, 63, 80, 28, 30, 58, 11, 44, 87, 22, 54, 74, 10, 55, 28, 46, 29, 10, 5072, 34, 26, 25, 8, 51, 13, 30, 35, 8, 50, 65, 6, 16, 2, 2178, 35, 13, 14, 23, 3, 26, 90, 86, 25, 56, 91, 13, 92, 25, 37, 570, 69, 98, 95, 45, 47, 29, 86, 28, 73, 44, 46, 65, 84, 96, 24, 12, 72, 68, 93, 57, 92, 52, 5, 2, 85, 63, 56, 55, 12, 85, 77, 39]
[2, 2, 2, 2, 3, 5, 5, 6, 8, 8, 9, 10, 10, 10, 11, 11, 12, 12, 13, 13, 13, 14, 16, 20, 22, 23, 24, 24, 25, 25, 25, 26, 26, 28, 28, 28, 29, 29, 30, 30, 34, 35, 35, 37, 39, 44, 44, 45, 46, 46, 47, 48, 50, 51, 52, 54, 55, 55, 56, 56, 57, 58, 63, 63, 64, 65, 65, 68, 69, 71, 72, 73, 74, 74, 77, 80, 84, 84, 85, 85, 86, 86, 87, 90, 91, 91, 92, 92, 93, 95, 96, 98, 570, 2178, 5072]
2
5072
